utils/oscap-bootc.8

```diff
@@ -0,0 +1,29 @@
+.TH oscap-bootc "8" "November 2024" "Red Hat, Inc." "System Administration Utilities"
+
+.SH NAME
+oscap-bootc \- Tool for building hardened bootable container images
+
+.SH DESCRIPTION
+The oscap-bootc tool is a convenience script that makes building hardened bootable container images easier.
+This tool is designed to be used during the build of the bootable container image.
+Include oscap-bootc in your Containerfile that will be used to build your bootable container image.
+The oscap-bootc runs oscap tool on a given container image.
+
+The oscap-bootc tool can't be used anywhere else than in a Containerfile.
+
+.SH USAGE
+
+Usage of the tool mimics usage and options of oscap(8) tool.
+
+oscap-bootc OSCAP_ARGUMENT [OSCAP_ARGUMENT...]
+
+Refer to oscap(8) to learn about OSCAP_ARGUMENT options.
```

Comment:
The script only supports certain options (specified in `parse_args` function) from `oscap(8)` so other options like `--rule`, `--skip-rule`, etc. won't work. Wouldn't it be better if we explicitly list only supported options in the man page?

So instead of
```
oscap-bootc OSCAP_ARGUMENT [OSCAP_ARGUMENT...]
```
we would have something like
```
oscap-bootc [--tailoring-file TAILORING_FILE]
    [--tailoring-id COMPONENT_REF_ID]
    [--results-arf FILE] [--report FILE]
    --profile PROFILE INPUT_DATA_STREAM
```